Core/Utilities/Graphs/GraphData.py

```diff
@@ -146,13 +146,13 @@ def sortLabels(self, sort_type='max_value', reverse_order=False):
         self.label_values = [self.plotdata.parsed_data[l] for l in self.labels]
     else:
       if sort_type == 'max_value':
-        pairs = zip(self.subplots.keys(), self.subplots.values())
+        pairs = list(zip(self.subplots.keys(), self.subplots.values()))
```

Comment:
This does not look correct. Did you mean
```suggestion
        pairs = list(zip(list(self.subplots), self.subplots.values())))
```
?